modules/gdscript/gdscript_editor.cpp

```diff
@@ -499,8 +499,14 @@ struct GDScriptCompletionIdentifier {
 
 static void _get_directory_contents(EditorFileSystemDirectory *p_dir, Set<String> &r_list) {
 
+#ifdef TOOLS_ENABLED
+	const String quote_style = EDITOR_DEF("text_editor/completion/use_single_quotes", 0) ? "'" : "\"";
```

Comment:
Please use `false` for booleans, not `0`.

Reply:
@akien-mga Right, I just had copied from another location on the same file where 0 was used as `bool` on  `EDITOR_DEF`. Does Godot has some type of guideline about style/formatting?

Reply:
There's http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html, but it doesn't cover this use case. Usually "do like surrounding code" is the proper guideline, but you lucked out here it seems :)